libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/trip/service/TripService.kt

```diff
@@ -9,7 +9,7 @@ import com.mapbox.navigation.core.trip.session.TripSession
 /**
```

Comment:
Let's remove the `TODO` ☝️

Reply:
This can be moved out of the `internal` package as it has the `internal` modifier now.

Reply:
fixed